[Add-ins] (on send) Update supported platforms (#636)
OfficeDev/outlook-dev-docs: docs/add-ins/outlook-on-send-addins.md

## Limitations

The on send feature currently has the following limitations.

> - **AppSource** &ndash; You can't publish Outlook add-ins that use the on send feature to [AppSource](https://appsource.microsoft.com). Add-ins that use the on send event will fail AppSource validation. Such add-ins should be installed by admins.

Reviewer: ```suggestion
- **AppSource** &ndash; You can't publish Outlook add-ins that use the on send feature to [AppSource](https://appsource.microsoft.com), as they will fail AppSource validation. Add-ins that use the on send feature should be deployed by administrators.
```

```powershell
Get-OWAMailboxPolicy OWAOnSendAddinAllUserPolicy | Set-OWAMailboxPolicy –OnSendAddinsEnabled:$false
```

### [Windows](#tab/windows)

Add-ins for Outlook on Windows that use the on send feature should run for any users who have them installed. However, if users are required to run the add-in to meet compliance standards, then the group policy **Disable send when web extensions can't load** must be set to **Enabled** on each applicable machine.

To set mailbox policies, administrators can download the [Administrative Templates tool](https://www.microsoft.com/download/details.aspx?id=49030) then access the latest administrative templates by running the Local Group Policy editor, **gpedit.msc**.

#### What the policy does

For compliance reasons, administrators may need to ensure that users cannot send email until the latest on send add-in is available to run. Administrators must enable the group policy **Disable send when web extensions can't load** so that all add-ins are updated from Exchange and available to verify each email meets expected rules and regulations on send.

|Policy status|Result|
|---|---|

> |Disabled.|Send allowed. Email can be sent without running the on send add-in, even if the add-in has not been updated from Exchange yet.|

Reviewer: Do the words "Disabled" and "Enabled" in the first column of each row need the period after the word?

The following are the parameters for the **addAsync** method:

> - *NoSend* &ndash; A string that is a developer-specified key to reference a notification message. You can use it to modify this message later. The key can’t be longer than 32 characters.

Reviewer: Suggest formatting all parameter names as code (e.g., `onSend`) instead of italics -- to prevent parameter names from being localized.
